developer/reference/components/makeswift-component.mdx

```diff
@@ -207,3 +211,49 @@ The header should now be visible on each page within the Visual Builder.
     alt="Header component showing on a page in the Makeswift Visual Builder"
   />
 </Frame>
+
+### Adding a description
+
+Then in this `layout.tsx' we can add the description field using markdown formatting.
+
+```tsx @/app/layout.tsx
+...
+
+          <MakeswiftComponent
+            snapshot={myHeaderSnapshot}
+            label={`Site Header`}
+            type={HEADER_COMPONENT_TYPE}
+            description = {
```

Comment:
Can we put the markdown in a separate variable to make this a bit more readable?